lib/charms/data_platform_libs/v0/database_provides.py

```diff
@@ -0,0 +1,316 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
It might be worth documenting in the PR description which files are worth reviewing in-depth and which aren't

Reply:
discussed offline - we can safely ignore any lib files. sorry for the bloated PR